src/pages/overview/pdf-services-api/howtos/split-pdf.md

```diff
@@ -33,34 +33,43 @@ Please refer the [API usage guide](../api-usage.md) to understand how to use our
   
      public static void main(String[] args) {
          try {
-             // Initial setup, create credentials instance.
-             Credentials credentials = Credentials.servicePrincipalCredentialsBuilder()
-                    .withClientId("PDF_SERVICES_CLIENT_ID")
-                    .withClientSecret("PDF_SERVICES_CLIENT_SECRET")
-                    .build();
-  
-             // Create an ExecutionContext using credentials and create a new operation instance.
-             ExecutionContext executionContext = ExecutionContext.create(credentials);
-             SplitPDFOperation splitPDFOperation = SplitPDFOperation.createNew();
-  
-             // Set operation input from a source file.
-             FileRef source = FileRef.createFromLocalFile("src/main/resources/splitPDFInput.pdf");
-             splitPDFOperation.setInput(source);
-  
-             // Set the maximum number of pages each of the output files can have.
-             splitPDFOperation.setPageCount(2);
-  
-             // Execute the operation.
-             List result = splitPDFOperation.execute(executionContext);
-  
-             // Save the result to the specified location.
-             int index = 0;
-             for (FileRef fileRef : result) {
-                 fileRef.saveAs("output/SplitPDFByNumberOfPagesOutput_" + index + ".pdf");
-                 index++;
-             }
-  
-         } catch (IOException| ServiceApiException | SdkException | ServiceUsageException e) {
+            // Initial setup, create credentials instance
+            Credentials credentials = new ServicePrincipalCredentials(
+                    System.getenv("PDF_SERVICES_CLIENT_ID"),
+                    System.getenv("PDF_SERVICES_CLIENT_SECRET"));
+
+            // Creates a PDF Services instance
+            PDFServices pdfServices = new PDFServices(credentials);
+
+            // Creates an asset from source file and upload
```

Comment:
same

Reply:
As discussed, IG this is from already reviewed samples.